pkg/client/client.go

```diff
@@ -194,9 +194,8 @@ func (c *Client) Health(ctx context.Context) error {
 			})
 			cancel()
 			if err != nil {
-				err = fmt.Errorf("error sending health information: %w", err)
-				c.logger.SystemErr(errors.E(op, err))
-				return err
+				c.logger.Errorf("error sending health information: %w", err)
```

Comment:
Please use the `SystemErr` function, it has domain specific logic.

Reply:
done